docs/reference/configuration/keywords.rst

```diff
@@ -494,11 +494,11 @@ RUNPATH_FILE
 ------------
 .. _runpath_file:
 
-When running workflows based on external scripts it is necessary to 'tell' the
-external script in some way or another were all the realisations are located in
+When running workflows based on external scripts, it is necessary to 'tell' the
+external script where all the realisations are located in
 the filesystem. Since the number of realisations can be quite high this will
-easily overflow the commandline buffer; the solution which is used is therefore
-to let ERT write a regular file which looks like this::
+easily overflow the commandline buffer; the solution used is
+to let ERT write a regular file. It looks like this::
```

Comment:
```suggestion
easily overflow the commandline buffer; the solution is
to let ERT write a file which looks like this::
```

Reply:
Jus a small suggestion :)

Reply:
Thanks Eivind. I would keep as it is just because I am too lazy to make a rebase on web version of Github :-)